docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -130,70 +130,178 @@ The remote control framework also allows mobile applications to send simulated b
 
-### Checking Permissions
-Prior to using any remote control RPCs, you must check that the head unit has the remote control capability. As you will encounter head units that do *not* support it, this check is important. To check for this capability, use the following call:
+### Multiple Modules (RPC v6.0+)
+Each module type can have multiple modules in RPC v6.0+. In previous versions, only one module was available for each module type. A specific module is controlled using the unique id assigned to the module. When sending remote control RPCs to a v6.0+, the `moduleInfo.moduleId` must be stored and provided to control the desired module. If no `moduleId` is set, the HMI will use the default module of that module type. When connected to <6.0 systems, the `moduleInfo` struct will be @![iOS]`nil`!@@![android, javaSE, javaEE]`null`!@, and only the default module will be available for control.
 
-If you do have permission to use the remote control feature, the capability object will have a list of @![iOS]`SDLButtonCapabilities`!@ @![android, javaSE, javaEE]`ButtonCapabilities`!@ that can be obtained via the `buttonCapabilities` property.
+### Getting Remote Control Module Information
+Prior to using any remote control RPCs, you must check that the head unit has the remote control capability. As you will encounter head units that do *not* support remote control, or head units that do not give your application permission to read and write remote control data, this check is important.
+
+When connected to head units supporting RPC v6.0+, you should save this information for future use. The `moduleId` contained within the `moduleInfo` struct on each capability is necessary to control that module.
 
 @![iOS]
 ##### Objective-C
 ```objc
-[self.sdlManager.systemCapabilityManager updateCapabilityType:SDLSystemCapabilityTypeRemoteControl completionHandler:^(NSError * _Nullable error, SDLSystemCapabilityManager * _Nonnull systemCapabilityManager) {
-    if (systemCapabilityManager.remoteControlCapability == nil || error != nil) {
-        // Remote control is not supported or capability could not be retrieved
-        return;
-    }
+[self.sdlManager.systemCapabilityManager subscribeToCapabilityType:SDLSystemCapabilityTypeRemoteControl withBlock:^(SDLSystemCapability * _Nonnull capability) {
+    if(!capability.remoteControlCapability) { return; }
+    <#Save remote control capabilities#>
+}];
+```
 
-    SDLRemoteControlCapabilities *capabilities = systemCapabilityManager.remoteControlCapability;
-    <#Code#>
+##### Swift
+```swift
+sdlManager.systemCapabilityManager.subscribe(toCapabilityType: .remoteControl, with: { capability in
+    guard capability.remoteControlCapability != nil else { return }
+    <#Save remote control capabilities#>
+})
+
+```
+!@
+
+@![java,javaEE,javaSE]
+```java
+// ToDo - Add example
+```
+!@
+
+#### Getting Module Data Location and Service Areas (RPC v6.0+)
```

Comment:
We can link to the header for **Setting the User's Seat** according to the SDL markdown spec (https://github.com/smartdevicelink/sdl_markdown_spec#header-anchor)

[Setting the User's Seat](#-setting-the-user's-seat)

Reply:
I'll give it a shot